src/ostorlab/serve_app/types.py

```diff
@@ -735,10 +718,10 @@ class OxoLinkInputType(graphene.InputObjectType):
 
 
 class OxoAssetInputType(graphene.InputObjectType):
-    android_file = OxoAndroidFileAssetInputType()
-    ios_file = OxoIOSFileAssetInputType()
-    android_store = OxoAndroidStoreAssetInputType()
-    ios_store = OxoIOSStoreAssetInputType()
+    android_apk_file = graphene.List(OxoAndroidFileAssetInputType)
+    ios_file = graphene.List(OxoIOSFileAssetInputType)
```

Comment:
Isn't this ios_ipa_file?

Reply:
Nope, its ios_file: 
![image](https://github.com/Ostorlab/oxo/assets/144013278/48c04f4e-6203-4037-9af7-9a50ccd4a1d5)